checker/manual/fenum-checker.tex

```diff
@@ -115,18 +113,46 @@ \section{Running the Fenum Checker\label{fenum-running}}
 
 \begin{itemize}
   \item
-If you define your own annotation, provide the name of the annotation using the
-\code{-Aquals} option:
+If you define your own annotation(s), provide the name(s) of the annotation(s)
+through the \code{-Aquals} option, using a comma-no-space-separated notation:
 
 \begin{alltt}
-  javac -processor org.checkerframework.checker.fenum.FenumChecker
-        \textit{-Aquals=myproject.qual.MyFenum} MyFile.java ...
+  javac -Xbootclasspath/p:\textit{/full/path/to/myProject/bin}:\textit{/full/path/to/myLibrary/bin} \ttbs
+        -processor org.checkerframework.checker.fenum.FenumChecker \ttbs
+        -Aquals=\textit{myModule.qual.MyFenum} MyFile.java ...
```

Comment:
Please add a test for this and the new -AqualDirs options. (Probably needs to be a jtreg test.  See checker/jtreg/READMD.)